docs/installation/index.md

```diff
@@ -35,6 +35,7 @@ The following ports need to be open to external connections:
 
 * 80 TCP - HTTP, used for provisioning purposes
 * 443 TCP - HTTPS, used for web access and api communication
+* 5647 TCP - qdrouterd - used for client and capsule actions
 * 5671 TCP - qpid ssl - used for client actions
```

Comment:
Would you mind removing this line, too? I don't think we need this one anymore for external connections, everything happens through the dispatch router.